packages/runtime-common/realm.ts

```diff
@@ -804,6 +805,36 @@ export class Realm {
     });
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I'm unclear why you needed to add this eslint disable, our lint rules should be setup such that params prefixed with `_` are allowed to be unsued.

Reply:
Fixed this eslint config